WPMedia/fusion-news-theme-blocks
# SENG-65 added video promo block

## Description
Added video promo block

## Jira Ticket
- https://arcpublishing.atlassian.net/browse/SENG-65

## Acceptance Criteria
PageBuilder users can configure a video (either VOD or Livestream) to appear on their homepage (or similar landing pages). They can also configure:
- Title
- Description
- LIVE toggle
- Autoplay

## Test Steps
1. This change is depending on this SDK PR https://github.com/WPMedia/engine-theme-sdk/pull/94
2. Add a new env variable `videoEnv` required for powa player (see below example)
3. In local env, link the SDK and the block then run local Fusion (ex https://github.com/WPMedia/Fusion-News-Theme/commit/ca61f84d0ce669b6e0849b3d3c6b92559bca3c3c)
4. Add the Video Promo block
5. Confgiure the video with content source (need to fetch video type item) or specify an available video UUID
6. Add title and description
7. Verify the video and content are showing up 

## Effect Of Changes
### Before
Doesn't exist

### After
![screencast 2020-09-23 17-41-33](https://user-images.githubusercontent.com/36861405/94079821-3dc59e00-fdc4-11ea-969e-9f7d00877362.gif)


## Dependencies or Side Effects
https://github.com/WPMedia/engine-theme-sdk/pull/94

## Review Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).
